get-pymodbus.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+git clone -b v3.1.0 https://github.com/pymodbus-dev/pymodbus
```

Comment:
could we have done this via `requirements.txt` file?

Reply:
Hmm I didn't try that, I assumed that the pip installation in a virtual environment would be less suitable for a docker build. I'll give it a go

Reply:
no, if we just install pymodbus via pip without the requirements.txt from the source code, it's missing several deps and `pymodbus.server` fails to execute. We would end up recreating most of the requirements.txt in the Dockerfile